src/app/creator-profile-page/creator-profile-top-card/creator-profile-top-card.component.html

```diff
@@ -18,6 +18,13 @@
         <i class="fas fa-ellipsis-h"></i>
       </a>
       <div class="dropdown-menu dropdown-menu-right p-0 border background-color-light-grey fs-15px" *dropdownMenu>
+        <a
+          class="dropdown-menu-item d-block link--unstyled p-10px feed-post__dropdown-menu-item"
+          (click)="reportUser()"
+        >
+          <i class="fas fa-flag"></i>
+          Report User
+        </a>
```

Comment:
hmm looks like you branched off your report users branch. let's try to keep these separate.